test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -50,4 +50,14 @@
             {Get-Verb -Group $group} | Should Not Throw
         }
     }
+
+    It "Should have descriptions for all verbs" {
+        $noDesc = Get-Verb | Where-Object { [string]::IsNullOrEmpty($_.Description) }
+        $noDesc | Should be $null
```

Comment:
For "all verbs" we should _count_.

Reply:
You mean I should test for `$verbsWithDescription.count | Should be $allVerbs.count` or do you mean something else?

Reply:
Yes. We should count verbs and verbs with Description.